ynput/OpenPype
# Burnins: Refactored burnins script

## Changelog Description
Refactored list value for burnins and fixed command length limit by using temp file for filters string.

## Additional info
Burnin script does not print irrelevant warning about not supported subdictionary formattings (it is supported). List values can have multiple list keys in single template and can expect other keys to fill (except timecode). FFmpeg filters are not passed to cli but through temp file and using `-filter_script` command to the file which should fix possible limit of command length.

## Testing notes:
1. Burnins should work with templates using dictionary subkeys (e.g. `{project[name]}`)
2. Burnins should work as expected with list values (currently supported only in focal length in maya)
    - make sure one of the burnin templates has `{focalLength}`
    - set keys for camera focal lenght in scene
    - publish
    - validate the burnin has right focal lenght values
